Task3/main.py

```diff
@@ -0,0 +1,70 @@
+# ЗАДАЧА-1
+# Написать свой декоратор который будет проверять остаток от деления числа 100 на результат работы функции ниже.
+# Если остаток от деления = 0, вывести сообщение "We are OK!», иначе «Bad news guys, we got {}» остаток от деления.
+import functools
+
+
+def decorator(func):
```

Comment:
It would be better to call decorator with another name. The name of the function should represent the action it does? like - check_remainder(func)